core_lib/core_lib.pro

```diff
@@ -93,7 +93,8 @@ HEADERS +=  \
     src/movieexporter.h \
     src/miniz.h \
     src/qminiz.h \
-    src/activeframepool.h
+    src/activeframepool.h \
+    src/structure/copymultiplekeyframesdialog.h
```

Comment:
You mean the .h, .cpp and .ui file, I guess. How do I move them?
I would right click on the destination folder and pick "Add existing file...", and then delete them in core_lib, but is that the right procedure, and will the .pro file be updated automatically?

Reply:
Good question. If you’re going to do it in Qt Creator I’d try renaming (simply include directory separators in the new name as needed). Personally I always do stuff like that manually because I have developed a certain distrust of Qt Creator, but that’s a matter of preference.